zerver/migrations/0527_alter_alertword_id_alter_archivedreaction_id_and_more.py

```diff
@@ -0,0 +1,278 @@
+from typing import Any
+
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.operations.base import Operation
+
+
+class MigrateIdToBigint(Operation):
```

Comment:
Probably worth giving this migration a clearer name, like `0527_convert_most_ids_to_bigints`.